<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Optical Proximity Correction</string>
  </property>
  <widget class="QPushButton" name="pushBtnGo">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Start processing!</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditStatusReport">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>431</width>
     <height>211</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>20</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="ImageView" name="imageViewInputFile">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>600</width>
     <height>450</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>133</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Input GDS file location</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>431</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="lineEditInputFileFullPath"/>
    </item>
    <item>
     <widget class="QPushButton" name="pBtnSelectInputFile">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>201</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Pixel size in X</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Pixel size in Y</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Iterations</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Sigma</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Resolution dot width (px)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Required dose for a resolution dot (mJ/cm2)</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="lineEditPixelSizeInX"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditPixelSizeInY"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditIterations"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditSigma"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditDotWidthPx"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditDotDose"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>172</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QPushButton" name="pushBtnEstimateDose">
      <property name="text">
       <string>Estimate does (mJ/cm^2)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Estimated dose (mJ/cm^2)</string>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>31</y>
         <width>151</width>
         <height>38</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_DoseResLine">
          <property name="text">
           <string>Resolution lines: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_DoseTypPattern">
          <property name="text">
           <string>Typical patterns:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>371</width>
     <height>18</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Processed Input Pattern</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_currentProcessingLayer">
      <property name="text">
       <string>Current processing layer:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageView</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
